utils/task_executor.py

```diff
@@ -1,46 +1,45 @@
 import logging
 from utils import config, helpers
 from integrations import browser, computer, discord, facebook, google, lam_at_home, open_interpreter, telegram
+from utils import contact_parse
 
 def execute_task(context, text):
     words = text.split()
-    if len(words) < 2:
+    if len(words) < 3:
```

Comment:
this is changed back to three, not all the integrations take 3 words. like the lamathome terminate for example.